assignment5/unknown_language.py

```diff
@@ -0,0 +1,55 @@
+'''
+Created on 07 July 2017
+
+@author: Eva
+'''
+
+def get_alphabet_adj(dictionary):
+    ''' Creates an alphabet given the dictionary of an alien language.
+        
+        Args:
+            dictionary: a list of words from the alien language given in 
+            lexicographic order
+        
+        Returns:
+            The alphabet of the dictionary in lexicographic order.
+    
+    '''
+    if not is_valid(dictionary):
+        raise ValueError("must be non-empty: dictionary")
+    
+    alphabet = []
+    
+    for j in range(len(dictionary) - 1):
+        word_cur = dictionary[j]
+        word_next = dictionary[j+1]
+        
+        alphabet = check_adjacent_words(word_cur, word_next, alphabet)
+    
```

Comment:
I like the use of this helper function here - it really makes your code more understandable and breaks it into parts :blue_heart: